docs/howto/use-alternative-secrets-backend.rst

```diff
@@ -92,8 +92,13 @@ file ``/files/my_conn.json`` when it looks for connections.
 
 The file can be defined in ``JSON``, ``YAML`` or ``env`` format.
 
+Any extra json parameters can be provided using keys like ``extra_dejson`` and ``extra``.
+The key ``extra_dejson`` can be used to provide parameters as JSON object where as the key ``extra`` can be used in case of a JSON string.
+The keys ``extra`` and ``extra_dejson`` are mutually exclusive.
+
 The JSON file must contain an object where the key contains the connection ID and the value contains
 the definitions of one or more connections. The connection can be defined as a URI (string) or JSON object.
+
 For a guide about defining a connection as a URI, see:: :ref:`generating_connection_uri`.
 For a description of the connection object parameters see :class:`~airflow.models.connection.Connection`.
 The following is a sample JSON file.
```

Comment:
You can move this sentence for introduction. We have the first use of this concept there, so these links should be earlier.

Reply:
Moved it to the inro